### DISCUSSION TITLE
Welcome to jcl Discussions!

### DISCUSSION THREAD

**Discussion by obones** [project-jedi/jcl#117] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ronaldhoek** [comment]
I've been using Delphi for several years (25) and have created plenty of useful applications (mostly internal tools). But some are still live customer applications running in high-volume environments. 
In those years I started to use the JEDI libraries (JCL/JVCL) - at first as RxLib and later as the project JEDI library - and also contributed some parts to the code (new features en bug fixes) where possible.

However, after debating for a long time, we as a company moved to another environment (.NEt / Angular) to develop our new browser-based application.

But since my heart still lies with Delphi (and we still have some live Delphi applications), I will keep an eye on this community and keep contributing when possible!

**Comment by denistotoliciu** [comment]
Hello,
I have been programming in Pascal for quite some time out of passion and curiosity.  Have also worked with Delphi at several companies. Wanted to learn to give and had this idea of jumping into open source. So there they were: FreePascal and Lazarus and JCL.
Have programmed in other languages, but nothing can replace the one true first love: Pascal!

JCL impressed me right from the first moment I have seen it and it is a great marvel for all the Pascal programmers out there. 
The name Project JEDI is very cool and very inspired, too!

**Comment by jfudickar** [comment]
Hi,
i started my development career in 1987 with Turbo Pascal 3. Since then I've used all turbo pascal releases and Delphi from version 1:-)
I started with JCL/JVCL some years ago and put a lot off effort in the AppStore components, some database stuff and the DynControl components.
I'm developing for 20 years now an Oracle Support Tool, but now only in my free time.
And the free time is getting less and less and in my compnay I'm no longer working with oracle, so the need to enhance the tool is getting less and less.
And so now I'm mainly watching and fixing things I easily can fix.
Jens

**Comment by denistotoliciu** [comment]
Happy New Year!
I wish you many wonderful ideas and internal energy to fulfill them!
